File: 프로그래머스/lv0/120853. 컨트롤 제트/컨트롤 제트.py
def solution(s):
    s=list(s.split())
    print(s)
    answer = 0
    tmp=''
    for i in range(len(s)):
        if s[i] == 'Z':
            answer -= int(s[i-1])
        else:
            answer+= int(s[i])
    return answer

File: 프로그래머스/0/120853. 컨트롤 제트/컨트롤 제트.java
class Solution {
    public int solution(String s) {
        int answer = 0;
        int tmp = 0;
        String[] st = s.split(" ");
        for(int i=0; i<st.length; i++){
            if(st[i].equals("Z")){
                answer-=tmp;
                tmp = 0;
            }else{
                tmp = Integer.parseInt(st[i]);
                answer += tmp;
            }
        }
        return answer;
    }
}